translations/tiled_de.ts

```diff
@@ -593,17 +593,17 @@
     <message>
         <location line="+8"/>
         <source>&amp;Never</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Nie</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>For &amp;Selected Objects</source>
-        <translation type="unfinished"></translation>
+        <translation>Für ausge&amp;wählte Objekte</translation>
```

Comment:
I was under the impression that the accessor key was generated from the ampersand in the text and not hardcoded? @bjorn how do you handle it when the original key is not present in the translation? is there a real value in having the same accessor key in all languages, and does it trump the value of having memorizable/sensible accessors?